tests/react-test-app/package.json

```diff
@@ -6,8 +6,9 @@
     "start": "react-native start"
   },
   "dependencies": {
-    "react": "15.3.2",
-    "react-native": "^0.37.0",
+    "invariant": "^2.2.2",
```

Comment:
Yes. Though it's not a direct dependency, it seems to be the way people resolve it in https://github.com/facebook/react-native/issues/11327. Maybe there is a nicer solution, this is a bit hacky..